dotnet/src/Connectors/Connectors.Memory.Postgres/PostgresConstants.cs

```diff
@@ -64,4 +74,15 @@ internal static class PostgresConstants
     /// <summary>The name of the column that returns distance value in the database.</summary>
     /// <remarks>It is used in the similarity search query. Must not conflict with model property.</remarks>
     public const string DistanceColumnName = "sk_pg_distance";
+
+    /// <summary>The default index kind.</summary>
+    public const string DefaultIndexKind = IndexKind.Hnsw;
```

Comment:
I think it's also OK to use Flat here as the default.  When I was suggesting using Hnsw earlier, I didn't realise that postgres can also work without, and that's a totally acceptable default.